src/index.js

```diff
@@ -62,21 +63,24 @@ const maybeReverseCondition = (pathToCheck, condition, attributes) => {
 };
 
 /**
- * Validate a policy.
+ * Validate value with AJV.
  *
- * @param {object} policy the policy to validate
- * @returns {boolean} true iff the policy is valid
- * @throws Error if the policy is invalid
+ * @param {string} schemaName the name of the JSON schema to use for validation.
+ * @param {object} value the value to validate.
+ * @returns {boolean} true if the value is valid based on specified schema.
+ * @throws Error if the the value is invalid based on the specified schema.
  */
-const validate = (policy) => {
-  const valid = ajv.validate('Policy', policy);
+const validateJsonSchema = curry((schemaName, value) => {
```

Comment:
what's the `curry` part necessary for?

Reply:
So we can reuse this and partially apply `Policy` to still have the same `validate` function that we had before, while still being able to use it for validating reduce options.